.stylelintrc

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
These are the same rules used in [our other project, Obachine.])https://github.com/rijkvanzanten/obachine/blob/master/.stylelintrc)